### DISCUSSION TITLE
Implement scroll bars on Sideband

### DISCUSSION THREAD

**Discussion by hardBSDk** [markqvist/Reticulum#135] [Requests for New Features]
No scrollbars available.

**Comment by markqvist** [comment]
If there is a way to do this in the KivyMD framework, it should be quite simple. If not a scrollbar class has to be written from scratch. I think it makes most sense to let them appear on desktop only, and stay invisible on mobile.

**Reply by hardBSDk** [reply]
https://kivy.org/doc/stable/api-kivy.uix.scrollview.html
https://www.geeksforgeeks.org/python-scrollview-widget-in-kivy/
https://kivy.org/doc/stable/api-kivy.uix.slider.html
https://github.com/kivy/kivy/wiki/A-draggable-scrollbar-using-a-slider
